fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDeviceHL.xtend

```diff
@@ -208,15 +208,35 @@ __all__ = ["«cls.name»", "main"]
     //    Constructors
     //====================================================
     def pythonConstructors(PogoDeviceClass cls)  '''
-«IF true»«cls.constructorHL»
+«IF true»    def init_device(self):
+        """ Initializes the attributes and properties of the «cls.name»."""
```

Comment:
- There shouldn't be a leading space in the docstring: ```"""Initializes the ...```
- As in previous review, please use British spelling:  "Initialises"

Reply:
Done. Link to the commit
https://github.com/ska-telescope/pogo/commit/60b7ef2651fc88fc6d518d15e00c8fa216d384a4